packages/browser/test/e2e/test/breadcrumbs.test.js

```diff
@@ -0,0 +1,604 @@
+const { test, expect } = require('@playwright/test');
```

Comment:
Stuck with JS at the moment, because [this part](https://github.com/getsentry/sentry-javascript/blob/onur/playwright-browser-tests/packages/browser/test/e2e/test/utils/helpers.js#L22) is prone to get broken when the injected function is compiled TS. Might move the functions into separate JS files, and convert test files to TS. But IMO it might cost us clarity. I'm still on the lookout to find a better way to inject code.

Reply:
Can we pre-construct the pages? So that there are some pre-defined fixtures that we just reference by name in the test?

Reply:
Then we don't have to inject anything, we just navigate to the page containing the current test.

Reply:
Yes, that's possible. Thinking of this, as we also have a set of `Sentry.init()` scenarios now, we can pre-construct pages with both scenarios and fixtures, even (if that makes sense) use a test matrix-like structure when needed. I can convert to structure to that, should not take too much effort.